Components/buttons/button.html

```diff
@@ -0,0 +1,85 @@
+            <div class="version"> v1.0</div>
+            </nav>
+            <div class="item-container">
+                <div class="install-ui"><a href="/installGroot.html/install.html" class="install-text"> Installation </a></div>
+                        <a href="/Components/avatar/avatar.html" class="avatar-text ">Avatars</a>
+                        <a href="/Components/alert/alert.html" class="avatar-text">Alert</a>
+                        <a href="/Components/badges/badges.html" class="avatar-text">Badges</a>
+                        <a href="/Components/buttons/button.html" class="avatar-text">Buttons</a>
+                        <a href="/Components/card/card.html" class="avatar-text">Cards</a>
+                        <a href="/Components/modal/modal.html" class="avatar-text">Modals</a>
+                        <a href="/Components/input/" class="avatar-text">Input</a>
+                </div>
+                </div>
+        <div class="groot-container second">
+            <div class="avatar-container">
+                <span style="font-size: 2.5rem;">Button</span><br/>
+                <span style=><p style="font-size: 1.3rem; color: grey;border-left: 3px solid #6B7280;margin-left: 1rem;">The Button component is used to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.</p></span>
+            </div>
+            
+            <span style="font-size: 2rem; margin-left: 3rem; color: grey;">Button Primary</span><br/>
+            <div class="avatar-box">
+            <div class="badges">
+                <button class="button-first button-first-color"> ADD TO BAG </button>
+                <button class="button-third button-third-color"> CONFIRM </button>
+            </div>
+        </div>
+        <div class="badges-code">
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=sharp&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522button-first%2520button-first-color%2522%253E%2520ADD%2520TO%2520BAG%2520%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522button-third%2520button-third-color%2522%253E%2520CONFIRM%2520%253C%252Fbutton%253E"
+            style="width: 620px; height: 127px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+        
+        <div class="avatar-container">
+            <span style="font-size: 2rem;  color: grey;">Button Link</span><br/>
+            <div class="avatar-box">
+                <div class="badges icon">
+                    <button class="button-second button-second-color"> WISHLIST </button>
+                    <button class="button-fourth"> RESET </button>
```

Comment:
Hovering over 'Wishlist' button changes the cursor to pointer but hovering over 'Reset' button doesn't. Fixing this would add consistency.

Reply:
Thank you for your feedback. I will definitely do the changes mentioned by you.